frontend/src/metabase/admin/datamodel/metadata/components/MetadataFieldSettings/MetadataFieldSettings.tsx

```diff
@@ -208,6 +208,15 @@ const FieldBreadcrumbs = ({
 
 // eslint-disable-next-line import/no-default-export -- deprecated usage
 export default _.compose(
+  Fields.load({
```

Comment:
This fixes a crash.
The order matters here.

Reply:
what kind of crash?

Reply:
Reading property of an undefined object. I didn't find the root cause.
It made 1 test fail (I can't remember which one, but it's easy to find out :)).